File: server/CinemasService/Errors/CinemaErrors.cs
namespace CinemasService.Errors
{
    public static class CinemaErrors
    {
        public static class Create
        {
            public const string NameRequired = "name_required";
            public const string NameMaxLength = "name_max_length";
            public const string NameAlreadyExists = "name_already_exists";


            public const string LocationRequired = "location_required";
            public const string LocationMaxLength = "location_max_length";

            public const string DescriptionRequired = "description_required";
            public const string DescriptionMaxLength = "description_max_length";

            public const string CapacityInvalid = "capacity_invalid";
            public const string ImageUrlInvalid = "image_url_invalid";
        }

        public static class Update
        {
            public const string CinemaNotFound = "cinema_not_found";

            public const string NameMinLength = "name_min_length";
            public const string NameMaxLength = "name_max_length";

            public const string LocationMinLength = "location_min_length";
            public const string LocationMaxLength = "location_max_length";

            public const string DescriptionMinLength = "description_min_length";
            public const string DescriptionMaxLength = "description_max_length";

            public const string ImageUrlInvalid = "image_url_invalid";

            public const string CapacityInvalid = "capacity_invalid";

            public const string StatusInvalid = "status_invalid";

        }

        public static class Get
        {
            public const string CinemaNotFound = "cinema_not_found";
        }

        public static class Delete
        {
            public const string CinemaNotFound = "cinema_not_found";
        }

    }
}

File: ui/cms/src/validation/api-errors/CinemaErrors.ts
export const CinemaErrors: Record<string, string> = {
  name_required: "El nombre del cine es obligatorio.",
  name_max_length: "El nombre del cine es demasiado largo.",
  name_already_exists: "Ya existe un cine con este nombre.",

  location_required: "La ubicación del cine es obligatoria.",
  location_max_length: "La ubicación del cine es demasiado larga.",

  description_required: "La descripción del cine es obligatoria.",
  description_max_length: "La descripción del cine es demasiado larga.",

  capacity_invalid:
    "La capacidad del cine debe ser un número válido mayor a cero.",
  image_url_invalid: "La URL de la imagen proporcionada no es válida.",

  cinema_not_found: "No se encontró el cine solicitado.",

  name_min_length: "El nombre del cine es demasiado corto.",

  location_min_length: "La ubicación del cine es demasiado corta.",

  description_min_length: "La descripción del cine es demasiado corta.",

  status_invalid: "El estado proporcionado no es válido.",
};
